pkg/engine/apicall/executor.go

```diff
@@ -56,7 +56,7 @@ func (a *executor) executeK8sAPICall(ctx context.Context, path string, method ky
 	}
 	jsonData, err := a.client.RawAbsPath(ctx, path, string(method), requestData)
 	if err != nil {
-		return nil, fmt.Errorf("failed to %v resource with raw url\n: %s: %v", method, path, err)
+		return nil, fmt.Errorf("failed to %v resource with raw url\n: %s: %w", method, path, err)
```

Comment:
Per the [doc](https://pkg.go.dev/golang.org/x/xerrors#section-readme) from Go, both should work:
>Also, Errorf's %w verb was incorporated into fmt.Errorf.

Are you able to verify it?

Reply:
How would I do that? I assumed there would be a dev container pushed somewhere but I wasn't able to find one